tests/test_rfn_query.py

```diff
@@ -0,0 +1,138 @@
+import datetime as dt
+import pandas as pd
+from pandas.compat import lrange
+from pandas import DataFrame
+
+from bgbb.sql.sql_utils import run_rec_freq_spk, to_s3_fmt
+import pyspark
+from pytest import fixture
+
+
+spark = pyspark.sql.SparkSession.builder.appName("test").getOrCreate()
+
+
+def int2date(i):
+    return model_start_date + dt.timedelta(days=i - 1)
+
+
+model_start_date = dt.date(2018, 6, 1)
```

Comment:
Nit: all caps the global variables and place them between imports and the first function definition.